map.html

```diff
@@ -27,12 +27,12 @@
 
 <body>
   <div class="wrapper">
-    <div class="site-header" role="banner">
-      <div class="hero-section">
-        <div class="hero-section__headline">The impacts of climate change are not distributed equally.</div>
-        <img class="hero-section__image" src="/assets/images/woman-yellow-jacket.png" alt="woman yellow raincoat">
-      </div>
-    </div>
+    <header class="header">
+      <h1 class="header__title">Climate Vulnerability</h1>
+      <h2 class="header__subtitle">A MetroCommon <span aria-hidden class="header__metrocommon-word-mark">✕</span> 2050 Report</h2>
+
+      <div class="header__image"></div>
```

Comment:
Nice semantic HTML refactor! To clarify, are we scrapping the "The impacts of climate change are not distributed equally" headline? It's still there in the XD I'm referencing, but I could be out of date.